kiwi/boot/image/dracut.py

```diff
@@ -58,6 +62,75 @@ def include_file(self, filename, install_media=False):
             self.included_files_install.append('--install')
             self.included_files_install.append(filename)
 
+    def include_module(self, module):
+        """
+        Include module to dracut boot image
+
+        :param string module: module to include
+        """
+        if module not in self.modules:
+            self.modules.append(module)
+            self.modules_args = [
+                '--add', ' {0} '.format(' '.join(self.modules))
+            ]
+
+    def omit_module(self, module):
+        """
+        Omit module to dracut boot image
+
+        :param string module: module to omit
+        """
+        if module not in self.omit_modules:
+            self.omit_modules.append(module)
+            self.omit_modules_args = [
+                '--omit', ' {0} '.format(' '.join(self.omit_modules))
+            ]
+
+    def clear_modules_config(self):
+        """
+        Clears the modules configuration
+        """
+        self.modules = []
+        self.modules_args = []
+        self.omit_modules = []
+        self.omit_modiles_args = []
+
+    def persistent_config(self, config=None, config_file=None):
+        """
+        Writes modules configuration into a dracut configuration file.
+
+        If modules are not provided it uses the self modules attributes.
+        Contrary, if modules are provided as arguments it only uses the
+        provided ones.
+
+        :param dict config: a dictionary containing the modules to add and omit
+        :param string conf_file: configuration file to write
+        """
+        dracut_config = []
+        if config is None:
```

Comment:
python evaluates empty dicts as well as None type values as False. You can safely write

```
if not config:
```